sys/posix/pthread/pthread_spin.c

```diff
@@ -37,7 +39,7 @@ int pthread_spin_destroy(pthread_spinlock_t *lock)
         return EINVAL;
     }
 
-    (void) lock;
+    atomic_flag_clear(&(lock->flag));
```

Comment:
This is basically a semantic change (from doing nothing to doing something), right? So, I would propose to have a separate commit for this.